app/controllers/characters_controller.rb

```diff
@@ -37,13 +43,19 @@ def update
 
   def character_params
     params.require(:character).permit(
-                                        :name,
-                                        :user_id,
-                                        :game_id,
-                                        :birthday,
-                                        :hobby_1,
-                                        :hobby_2
-                                        )
+                                      :id,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the previous line.